### ISSUE TITLE
a11y: show that sorting was changed 

### ISSUE THREAD

**Issue by LiKang6688** [qlik-oss/sn-table#234]
when you have press space and changed the sorting,  information should show that sorting was changed.

**Comment by haxxmaxx** [comment]
You mean that the the screen reader should announce it? it is already showing, the arrow changes direction
